### ISSUE TITLE
How do I use the search?

### ISSUE THREAD

**Issue by 8ullyMaguire** [db0/pythorhead#61]
I've tried

```python
search_results = lemmy.search(post_url)
```

but I get 

```
AttributeError: 'Lemmy' object has no attribute 'search'
```

**Comment by db0** [comment]
We don't have a generic search currently. We do have `lemmy.discover_community()`

**Comment by NicKoehler** [comment]
> We don't have a generic search currently. We do have `lemmy.discover_community()`

I implemented it, maybe he have an old version?

**Comment by db0** [comment]
Oh right yes https://github.com/db0/pythorhead/blob/74f2fb4248d30416fe43af72f7dc63d83bba12af/pythorhead/lemmy.py#L52 forgot about that

**Comment by 8ullyMaguire** [comment]
Yeah you are right I'll try again.

**Comment by 8ullyMaguire** [comment]
You are right I hadn't upgraded sorry. This must be an API bug because I don't get any results even though it seems to be working.
